README.md

```diff
@@ -57,7 +56,7 @@ $client_secret = $oidc->getClientSecret();
 // Configure a proxy
 $oidc->setHttpProxy("http://my.proxy.com:80/");
 
-// Configure a cert
+// Configure a self-signed TLS certificate
```

Comment:
why self-signed only? can be any cert. I see no reason to change this comment .... besides to write `certificate` instead of `cert`